src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -178,6 +179,36 @@ object CollectDuplexSeqMetrics {
                        var fraction_raw_observations: Proportion = 0,
                        var fraction_unique_observations: Proportion = 0
                       ) extends Metric
+
+  /**
+    * Metrics produced by `CollectDuplexSeqMetrics` describing the set of observed duplex UMI sequences and the
+    * frequency of their observations.  The UMI sequences reported may have been corrected using information
+    * within a double-stranded tag family.  For example if a tag family is comprised of three read pairs with
+    * UMIs `ACGT-TGGT`, `ACGT-TGGT`, and `ACGT-TGGG` then a consensus UMI of `ACGT-TGGT` will be generated.
+    *
+    * UMI pairs are normalized within a tag family so that observations are always reported as if they came
+    * from a read pair with read 1 on the positive strand.  E.g. a read pair with UMI AAAA-GGGG and with R1 on
```

Comment:
IDK why but this took longer for me to remember/understand.  Once I heard in my head, "the UMI from the read on the positive strand comes before the UMI from the read on the negative strand" it clicked.  Can you re-read and see if you agree?